src/ui/actions/timeline.ts

```diff
@@ -535,7 +545,7 @@ export function setFocusRegion(
         { time: endTime, point: "" },
         p => p.time
       );
-      const end = endIndex > 0 ? state.timeline.points[endIndex - 1] : { point: "", time: endTime };
+      const end = endIndex > 0 ? state.timeline.points[endIndex] : { point: "", time: endTime };
```

Comment:
`endIndex - 1` is wrong, we want the point we would have inserted this *before*, not the point we would have inserted it *after*.